spec/controllers/sufia/admin/admin_sets_controller_spec.rb

```diff
@@ -33,6 +33,11 @@
         expect(response).to be_success
         expect(assigns[:admin_sets]).to be_kind_of Array
       end
+
+      it 'defines a presenter' do
+        get :index
+        expect(assigns[:presenter]).to be_kind_of Sufia::AdminSetPresenter
```

Comment:
Please combine the assertion into the previous test.  Otherwise we have the overhead of multip test/request setup for a single assertion. Our test suite is too slow as is.